packages/odyssey-react-mui/src/labs/UiShell/UiShell.tsx

```diff
@@ -94,6 +95,7 @@ const UiShell = ({
           <UiShellContent
             {...componentProps}
             appComponent={appComponent}
+            initiallyVisible={initiallyVisible}
```

Comment:
Because it's a list of sections that are invisible, I'd probably name this something more like `invisibleSections`.